tests/test_base.py

```diff
@@ -1,6 +1,26 @@
 # This is a dummy test file; delete it once the package actually has tests.
 
+import warnings
+
 
 def test_import():
     import qutip_cupy
     assert qutip_cupy.__version__
+
+
+def test_import_cupy():
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=DeprecationWarning)
+        import cupy
+    assert cupy.__version__
+
+
+def test_norm_cupy():
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=DeprecationWarning)
+        import cupy as cp
+    import numpy as np
+    x = cp.array([1, 2, 3])
+    y = cp.array([2, 3, 4])
+    z = x + y
+    assert (cp.asnumpy(z) == np.array([3, 5, 7])).all()
```

Comment:
should usually be `numpy.testing.assert_allequal()` or something like that, but I understand these are just dummy tests (this test is called `test_norm`, but doesn't even call `norm`!)

Reply:
I was going to call norm, but then decided I wanted to do something with two arrays. Tx for catching this. Fixed.